### PR TITLE
Update developer guide to start aspdotnet project

### PR DESCRIPTION
**Pull request by min-tomato**
- Add a `docker-compose.yml` file to set up and start an MSSQL Server.
- Add a `README.md` to provide a developer guide for installing, configuring, and starting the `ASP.NET` project.

### CODE REVIEW

File: aspnet-core/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Table of Contents
+
+1. [Requirements](#requirements)
+2. [Configuration](#configuration)
+3. [Installation](#installation)
+
+## Requirements
+
+- .NET SDK (version 8)
+- MSSQL Server 2022
```
Comment (datntdev0) [inline aspnet-core/README.md:14]: Should we add the instruction for the NodeJS - Angular folder?

File: aspnet-core/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+cd Government.Services.Migrator
+dotnet run
+```
+
+### 3. Run the Web Host
+
+After building and migrating, you can run the web host. Navigate to the `Government.Services.Host` folder and start the project:
+
+```bash
+cd Government.Services.Host
```
Comment (datntdev0) [inline aspnet-core/README.md:69]: `Government.Services.Host` should change to `Government.Services.Web.Host`

**Review by datntdev0 — COMMENTED** [review]
Consider to put the prefix `feature` into the branch name. This will help us setup the pipelines in later

File: aspnet-core/README.md (hunk 1 above)
Comment (min-tomato) [reply aspnet-core/README.md:14]: Sure, I think after this pull request is merged, I will continue adding the instructions for them.

**Pull request closed without merging**